yt_dlp/extractor/tver.py

```diff
@@ -1,77 +1,99 @@
 # coding: utf-8
 from __future__ import unicode_literals
 
+import re
 
 from .common import InfoExtractor
-from ..compat import compat_str
 from ..utils import (
     ExtractorError,
-    int_or_none,
-    remove_start,
     smuggle_url,
+    str_or_none,
     traverse_obj,
 )
 
 
 class TVerIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?tver\.jp/(?P<path>corner|episode|feature|lp|tokyo2020/video)/(?P<id>[fc]?\d+)'
-    # videos are only available for 7 days
+    _VALID_URL = r'https?://(?:www\.)?tver\.jp/(?:(?:lp|corner|series|episodes?|feature|tokyo2020/video)/)+(?P<id>[a-zA-Z0-9]+)'
+    # NOTE: episode/ is an old URL
+    _NEW_URL_COMPONENT = '|'.join(re.escape(f'/{x}/') for x in ('series', 'episodes'))
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www\.)?tver\.jp/(?:(?P<type>lp|corner|series|episodes?|feature|tokyo2020/video)/)+(?P<id>[a-zA-Z0-9]+)'
```
see next comment

Reply:
That's a bit confusing regex but confirmed it works as it should

Reply:
I just named the group. Didn't make any other change

Reply:
because group `path` can be matched twice for an URL